core/dbt/context/base.py

```diff
@@ -681,6 +683,30 @@ def diff_of_two_dicts(dict_a, dict_b):
                 dict_diff.update({k: dict_a[k]})
         return dict_diff
 
+    @contextmember
+    @staticmethod
+    def diff_of_two_dicts_no_lower(dict_a, dict_b):
```

Comment:
This function name seems not describe what the function does, the values in returned dict is lowered

Reply:
Also doesn't found this being used in the PR

Reply:
It looks like @emmyoop is using this in https://github.com/dbt-labs/dbt-bigquery/pull/212, since privileges are case-sensitive on BigQuery.

I think the right approach here would be to have a single `diff_of_two_dicts` method, which performs a case-insensitive _comparison only_, but still returns the keys in the original case (from `dict_a`).

Reply:
Resolved in debc867